app/Modules/Setup/Controllers/SetupController.php

```diff
@@ -75,7 +75,26 @@ public function prerequisites()
      */
     public function dbConfig()
     {
-        return view('setup.dbconfig');
+        // generate a list of available drivers
+        $drivers = array();
+        foreach (\PDO::getAvailableDrivers() as $driver) {
+            // check if driver is supported by laravel
+            if(config('database.connections.' . $driver) != null) {
+                $drivers[$driver] = $driver;
+            }
+        }
+
+        // prepare the database name if we are using the sqlite driver
+        $current_settings = config('database.connections.mysql');

```

Comment:
The comment just above refer to sqlite but you are getting the mysql config `database.connections.mysql`?
Is it normal?

Reply:
The configuration values are replicated across the drivers. I just use the MySQL one as it has all the fields that the user can edit. See the config/database.php file. I will add a comment to explain.